### DISCUSSION TITLE
Should Titles in index.md and README.md files be the same?

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#400] [Core Contributions] [answered]
In some packages, the README.md file has the same title as the index file - for example, [autora-novelty-sampler](https://github.com/AutoResearch/autora-novelty-sampler). In other packages, the title of the README.md file is something AutoRA ... but the title of the index.md file is different - for example, [autora-theorist-darts](https://github.com/AutoResearch/autora-theorist-darts). There are arguments for both (the README.md file dosen't show up on the AutoRA mother homepage), but I think we should make it consistent. My suggestion would be to change the title of the README.md files to the title of the index.md file.

**Comment by benwandrew** [comment]
sounds good to me, i also support consistency here

**Reply by benwandrew** [reply]
just to confirm, we're going with the way the novelty sampler is currently set up? that is, with both README.md and index.md having just a simple eponymous title (eg, Novelty Sampler, Differentiable Architecture Search, Bayesian Machine Scientist, etc...)?

**Comment by younesStrittmatter** [comment, marked as answer]
In the README.md file, we have decided to use the contribution title with the prefix "AutoRA.". For example, "AutoRA Novelty Sampler". 

In the index.md file, we will only use the contribution title without the prefix "AutoRA".  For example, "Novelty Sampler".
